styles/globals.css

```diff
@@ -34,6 +34,11 @@ a {
   --morseOutputCharFontSize: calc(2 * var(--morseInputCharFontSize));
   --morseInputCharGap: 2px; /* if the value is not rounded to 1px, gap widths are different, that looks bad */
   --morseLineGap: clamp(5px, 2vmin, 10px);
+  --morseRightClickAreaWidth: calc(
```

Comment:
Remove `morseRightClickAreaWidth` variable and use calculation directly in `morseVariantButtonSize`.